float64 battery_level
float64 current
int32 ADC1
int32 ADC2
int32 ADC3
int32 ADC4
float64 speed_front_left
float64 speed_front_right
float64 odometry_left
float64 odometry_right
int32 version
int32 relay1
